backend/api/services/nodeServices.py

```diff
@@ -173,8 +173,22 @@ def getRemotePosts(request, **kwargs):
     Request a single Remote Post
     """
     @staticmethod
-    def getSingleRemotePost():
-        pass
+    def getSingleRemotePost(request):
+        # get external node
+        extNodeReq = request.headers.get('X-Request-Host')
+        extNode = Node.objects.get(host_url=extNodeReq)
```

Comment:
maybe have an error check in case the node doesnt exist?

Reply:
will return `None` now if extNode does not exist

Reply:
Good stuff, good to merge I would say